pkg/newlog/cmd/newlogd.go

```diff
@@ -1256,17 +1265,25 @@ func checkWatchdogRestart(entry *inputEntry, panicStackCount *int, buf *[]byte,
 
 	if entry.source == "pillar" && strings.Contains(origMsg, "pillar;panic") && !strings.Contains(entry.content, "rebootReason") {
```

Comment:
With this change can we remove the grep in watchdog-report.sh for this pattern?

Reply:
yes. but I'll wait for the change that removes the rsyslogd, since one can still do Force-Use-Oldlog right now.

Reply:
OK